packages/manager/apps/zimbra/package.json

```diff
@@ -46,10 +47,22 @@
     "tailwindcss": "^3.3.3"
   },
   "devDependencies": {
+    "@jest/globals": "^29.7.0",
     "@ovh-ux/manager-vite-config": "^0.6.2",
-    "@playwright/test": "^1.34.3",
+    "@testing-library/dom": "^10.1.0",
+    "@testing-library/jest-dom": "^6.4.6",
+    "@testing-library/react": "^16.0.0",
+    "@testing-library/user-event": "^14.5.2",
+    "@types/jest": "^29.5.12",
+    "@types/react": "^18.2.55",
+    "@types/react-dom": "^18.2.19",
+    "@vitejs/plugin-react": "^4.2.1",
+    "@vitest/coverage-v8": "^1.6.0",
+    "jest": "27.x",
```

Comment:
since you are using vitest you can remove jest deps

Reply:
It's fixed